app/(client)/about-us/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Products() {
```

Comment:
```suggestion
export default function AboutUs() {
```
ไม่ต้องแก้ก็ได้ เราแค่บอกเฉยๆ เดี๋ยวเราไปแก้เอง